### PR TITLE
Do some chores

### PR DESCRIPTION
**Pull request by hanahmily**
* Introduce more strict checkers
  * Replace yarn with npm
  * Add node version checker
  * Bump up skywalking-eye

Signed-off-by: Gao Hongtao <hanahmily@gmail.com>

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please fix CI.

**Comment by lujiajing1126** [comment]
Why we have to use `yarn` over `npm`?

File: ui/tools/checkversion.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const fs = require("fs");
+
+const nvmVersion = fs.readFileSync(".node-version").toString().trim();
```
Comment (lujiajing1126) [inline ui/tools/checkversion.js:22]: I suppose we can impose version check by setting the `engine` field in `package.json`

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines

File: ui/.node-version [hunk 2]
```diff
@@ -0,0 +1 @@
+16.15.0
```
Comment (lujiajing1126) [inline ui/.node-version:1]: I don't think `.node-version` is a common dotfile for node version selection.

Probably `.nvmrc` is a better dotfile which can be read by [`nvm`](https://github.com/nvm-sh/nvm#nvmrc).

And we may use `lts/gallium` as the content of `.nvmrc`.

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]

**Comment by hanahmily** [comment]
> Why we have to use `npm` over `yarn`?

There are two motivations:

  * skywalking-eye opts to `npm` to generate license files.
  * UI contributors are using the element UI which prefers `npm`.

**Comment by lujiajing1126** [comment]
> > Why we have to use `npm` over `yarn`?
> 
> There are two motivations:
> 
> * skywalking-eye opts to `npm` to generate license files.
> * UI contributors are using the element UI which prefers `npm`.

Make sense. But I'm still confused with the `.node-version` file.

It there any toolkit which follows this dotfile?

File: ui/tools/checkversion.js (hunk 1 above)
Comment (hanahmily) [reply ui/tools/checkversion.js]: `engine` is better to place the version. I updated the script

File: ui/.node-version (hunk 2 above)
Comment (hanahmily) [reply ui/.node-version]: Replaced it with `package.json`

**Comment by hanahmily** [comment]
> > > Why we have to use `npm` over `yarn`?
> > 
> > 
> > There are two motivations:
> > 
> > * skywalking-eye opts to `npm` to generate license files.
> > * UI contributors are using the element UI which prefers `npm`.
> 
> Make sense. But I'm still confused with the `.node-version` file.
> 
> It there any toolkit which follows this dotfile?

I removed this file.

**Review by lujiajing1126 — APPROVED** [review]

**Pull request merged**